paper/manuscript.tex

```diff
@@ -48,132 +51,165 @@
-These notebooks are a mix of executable code cells and markup. An analysis performed in such a notebook can be appended as supporting information, along with the data, to an article. Readers can then replicate the exact data analysis, provided they have set up the same computing environment. Setting up the computing environment is simplified using the \conda\ package manager \cite{conda}, and an environment file (although other approaches are available). To that end the \Jupyter\ notebooks for the example analysis performed in this paper are appended as supporting information.
+Here, we outline a new reflectometry analysis package, \refnx\ (version number 0.1 is used in this paper \cite{refnx}), that helps address the reproducibility issue for the reflectometry community\footnote{We do not mean that other programs are irreproducible, rather that the information provided by scientists in journal articles is often lacking.} by creating a scripted analysis workflow that is readily published alongside the publication, such as we have done with this paper (see the Supporting Information). The \refnx\ Python package is specifically designed for use in \Jupyter\ notebooks \cite{Kluyver:2016aa}, which provide a literate programming environment that mixes executable code cells, rich documentation of the steps that were performed, and the computational output. By including the analysis as performed by the authors in such a notebook and appending it as supporting information along with the data, readers are empowered to replicate the exact data analysis and potentially extend the analysis, provided they have set up the same computing environment. Setting up the computing environment is simplified using the \conda\ package manager \cite{conda}, and an environment file (although other approaches are available).
 
-\refnx\ has a wide range of sophisticated modelling capabilities including: parameter constraints, description of prior probabilities on parameters using arbitrary probability distribution functions, modular construction of models using a variety of components that are user extensible, co-refinement of datasets, handling of patchy layers, a graphical user interface, resolution handling, Markov Chain Monte Carlo sampling of the posterior probability distributions for the parameters, model selection, etc.
+%TODO cite https://osf.io/h9gsd/ "Millman and P�rez, Developing open source scientific practice" ?
 
-The posterior probability distribution for a parameter set in Bayesian statistics, Equation~\ref{eqn:1}, is proportional to the product of the prior probability and the likelihood (or the sum of the log-probabilities).
+% FIXME next para can be deleted? (dupe)
```

Comment:
Do you mean that it's duplicated in the method section?

Reply:
Almost the same sentence with the list of features is in there 3 times, I think.